Examples/NOTP_internal_access_only/provider.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    citrixadc = {
+      source = "citrix/citrixadc"
```

Comment:
it is a better practice to provide the provider version against which you have tested this solution to avoid any incompatible code in the future releases.